taccsite_cms/templates/guides/getting_started.html

```diff
@@ -39,12 +39,11 @@ <h3 id="create-account">Creating an Account</h3>
         </div>
 
         <div class="col">
-          <p>
-            First, you will need to create a TACC account using your UT System
-            Institution Credentials. Please navigate to the sign up page and
-            select your institution at
-            <a href="https://portal.tacc.utexas.edu/utdr" target="_blank" >portal.tacc.utexas.edu/utdr</a>
-            .
+          <p>If you have UT System Institution Credentials, you may create a TACC account from those by first selecting your institution at
+            <a href="https://portal.tacc.utexas.edu/utdr" target="_blank" >portal.tacc.utexas.edu/utdr</a>.
+          </p>
+          <p>Otherwise, you may sign up for a new TACC account at
+            <a href="https://portal.tacc.utexas.edu" target="_blank" >portal.tacc.utexas.edu</a>.
```

Comment:
what about https://portal.tacc.utexas.edu/account-request ? or is it better to bring them to main page

Reply:
Link is now updated (e1e21d0) to this more direct link. Thank you.

_I think this proposed direct link has ample information and branding to serve as a starting point._